packages/nodes-base/nodes/EditImage/EditImage.node.ts

```diff
@@ -1234,15 +1228,23 @@ export class EditImage implements INodeType {
 						// Combine the lines to a single string
 						const renderText = lines.join('\n');
 
-						const font = options.font || operationData.font;
+						let font = options.font as string | undefined;
```

Comment:
we still need operationData.font
```suggestion
						let font = (options.font || operationData.font) as string | undefined;
```